docker-compose.yaml

```diff
@@ -161,6 +167,10 @@ services:
       - "30333:30333"
     volumes:
       - ./spec-data:/cereio/spec-data
-      - ./chain-data:/data
+      - type: volume
+        source: chainData
+        target: /chainData
     command: bash -c "/usr/local/bin/cere -l${LOG_LEVEL} --name ${NODE_NAME} --chain /cereio/spec-data/${SPEC_PATH}/customSpecRaw.json --pruning archive --telemetry-url 'wss://telemetry.polkadot.io/submit/ 0' --bootnodes /ip4/${BOOT_NODE_IP_ADDRESS}/tcp/30333/p2p/${NETWORK_IDENTIFIER} /ip4/${BOOT_NODE_IP_ADDRESS_2}/tcp/30333/p2p/${NETWORK_IDENTIFIER_2} --ws-port 9944 --rpc-methods=Unsafe --ws-external  --unsafe-ws-external --rpc-cors all --rpc-external"
-    
+
+volumes:
```

Comment:
Is it any limits on volume size by default?

Reply:
When the system can kill volume?

Reply:
There is no limit on volume size, but we can set the maximum size to avoid host machine running out of Memory